### PR TITLE
Update Fedora install docs

### PR DESCRIPTION
**Pull request by spuiuk**
Update to reflect the change in default to CentOs 8 and the introduction
of the make clean target to clear the old setup.

Signed-off-by: Sachin Prabhu <sprabhu@redhat.com>

### CODE REVIEW

**Comment by anoopcs9** [comment]
retest this please

File: docs/Fedora-Setup.md [hunk 1]
```diff
@@ -8,11 +8,6 @@ Modify the steps accordingly for your setup.
 Ensure the following packages are installed
 > $ sudo dnf install qemu-kvm qemu-img git vagrant vagrant-libvirt ansible make libvirt-client
 
-Download and cache vagrant centos7 image.
-```
-$ vagrant box add "https://vagrantcloud.com/centos/7" --provider "libvirt"
-```
-
 For Fedora 33, we have to enable the use_session variable for vagrant libvirt vms to run properly. To do this, create a file ~/.vagrant.d/Vagrantfile with the following content
```
Comment (anoopcs9) [inline docs/Fedora-Setup.md:11]: I hope this is applicable to Fedora 34 also.

**Review by anoopcs9 — CHANGES_REQUESTED** [review]

**Comment by anoopcs9** [comment]
retest this please

File: docs/Fedora-Setup.md (hunk 1 above)
Comment (spuiuk) [reply docs/Fedora-Setup.md:11]: yes it is.

Comment (anoopcs9) [reply docs/Fedora-Setup.md:11]: I hope we can then update the line accrodingly.

Comment (spuiuk) [reply docs/Fedora-Setup.md:11]: This was written for and tested with Fedora 33. There is no reason why it shouldn't work for Fedora 34 but I haven't actually tested it with F34. Therefore for now, I would like to keep the docs at F33 as it is the version I tested against.

**Pull request merged**